tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -315,6 +315,11 @@ internal static partial class ConfigurationKeys
         /// <seealso cref="TracerSettings.ExpandRouteTemplatesEnabled"/>
         public const string ExpandRouteTemplatesEnabled = "DD_TRACE_EXPAND_ROUTE_TEMPLATES_ENABLED";
 
+        /// <summary>
+        /// Configuration key for enabling computation of metrics on the tracer side
```

Comment:
Let's clarify which metrics. There's are now 3 settings for "metrics".
```suggestion
        /// Configuration key for enabling computation of trace metrics (aka stats) on the tracer side
```